remotes/docker/authorizer.go

```diff
@@ -284,7 +284,13 @@ func (ah *authHandler) doBearerAuth(ctx context.Context) (token string, err erro
 				// Registries without support for POST may return 404 for POST /v2/token.
 				// As of September 2017, GCR is known to return 404.
 				// As of February 2018, JFrog Artifactory is known to return 401.
-				if (errStatus.StatusCode == 405 && to.Username != "") || errStatus.StatusCode == 404 || errStatus.StatusCode == 401 {
+				// As of December 2019, Harbor is known to return 403.
+				// As of January 2021, Docker community's registry implementation (https://github.com/docker/distribution)
```

Comment:
https://github.com/docker/distribution is now https://github.com/distribution/distribution

Reply:
Actually, it will be redirected to https://github.com/distribution/distribution automatically.
However, I think we'd better keep it updated. thx for pointing out.